kontroller/qml/ServerSettingsPage.qml

```diff
@@ -313,13 +310,13 @@ to make the remote control the amplifier volume.")
                 id: serverWakeUpPlugin
                 anchors.left: parent.left
                 anchors.right: parent.right
-                label: qsTr("WakeUp plugin")
+                label: qsTr("Wake-on-LAN plugin")
```

Comment:
Renamed this same as the README.

Reply:
This one should stay as "WakeUp" ("Wake-up" ?). The idea is that plugins can be implemented to use other mechanisms to wake the device, such as wi-fi plugs, for example. Wake-on-LAN is just one specific way (thus, a plugin) to do it, and currently the only implemented.